components/helper/EligibilityButton.tsx

```diff
@@ -0,0 +1,74 @@
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { Button } from '@mui/material';
+import React from 'react';
+
+export default function EligibilityButton() {
+  return (
+    <Button
+      href="/calculator/head-initial-1-cont"
+      variant="contained"
+      color="tertiary"
+      size="small"
+      className="calc-btn"
+      aria-label="Access our eligibility calculator"
+      sx={{
+        whiteSpace: 'nowrap',
+        py: 1,
+        px: 3,
+        color: 'text.secondary',
+        '&:hover': {
+          backgroundColor: 'tertiary.light',
+        },
+        '&:active': {
+          backgroundColor: '#FF5F40',
```

Comment:
I think this works for now because #FF5F40 color is only displayed when the eligibility button is actively pressed by the user. This color behavior is encapsulated inside the function and will be consistent across the website.